app/controllers/chores_controller.rb

```diff
@@ -0,0 +1,56 @@
+class ChoresController < ApplicationController
+  def create
+    title = params[:title]
+    description = params[:description]
+    due_date = params[:due_date]
+    assignee_email = params[:assignee_email]
+    hideout_id = params[:hideout_id]
+
+    if title.nil? || hideout_id.nil?
+      render status: 400
+    else
+      if assignee_email.nil?
+        assignee_id = nil
+      else
+        assignee_id = User.find_by(email: assignee_email).id
+      end
+      chore = Chore.create(title: title, description: description, due_date: due_date, assignee_id: assignee_id, hideout_id: hideout_id)
+      chore.save
+      render status: :created, :json => {title: title, description: description, due_date: due_date, assignee_email: assignee_email, hideout_id: hideout_id}
```

Comment:
We should be consistent with the status values. Let's just use integer codes instead of the symbols, so just replace the `status: :created` or `status: :symbol` stuff with `status: <insert_integer_code>`. For `:created`, it should be `201`.